challenges/objects-arrays.js

```diff
@@ -51,23 +79,49 @@ const graduates = [
 /* Request 1: Create a new array called universities that contains all the universities in the graduates array. This will be an array of strings.
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
-console.log(universities);
+
+let universities = [];
+let UniversitySorted = [];
+for (let i = 0; i < graduates.length; i++){
+
+    let university = graduates[i].university
+    universities.push(university)
+}
+UniversitySorted = universities.sort()
+
+console.log(UniversitySorted);
+
 
 /* Request 2: Create a new array called contactInfo that contains both first name and email of each student. This will be an array of strings.
 
 The resulting contact information strings should have a space between the first name and the email, like this: 
 "Josh josh@example.com"
 
 Log the result of your new array. */
-const contactInfo = [];
+let contactInfo = [];
+for (let i = 0; i < graduates.length; i++ ){
+  let contact = `${graduates[i].first_name} ${graduates[i].email}`
+  contactInfo.push(contact)
+};
+
 console.log(contactInfo);
 
+
+
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called unisWithUni that contains them all. This will be an array of objects. Log the result. */
-const unisWithUni = [];
+let unisWithUni = [];
+for (let i = 0; i <graduates.length; i++){
+  let uni = graduates[i].university
+  if (uni.includes("Uni")){
+  unisWithUni.push(graduates[i])
+}
+}
+
 console.log(unisWithUni);
```

Comment:
- wants you to return an array of objects with the university name --> { university: 'Universitat Rovira I Virgili Tarragona' }
-what would you change to return only that information?